bittensor/_metagraph/metagraph_impl.py

```diff
@@ -429,7 +429,7 @@ def sync ( self, block: int = None, cached: bool = True ) -> 'Metagraph':
         logger.success(self.subtensor)
         if block == None:
             block = self.subtensor.get_current_block()
-            if cached and self.subtensor.network in ("nakamoto", "local"):
+            if cached and self.subtensor.network in ("nakamoto", "local", "endpoint"):
```

Comment:
This would automatically assume Nakamoto, even though the endpoint might be connecting nobunaga or other networks. Do we want that?

Reply:
not sure how you mean, why would it automatically assume Nakamoto? if you put in Nobunaga it will sync manually using Nobunaga.